server/app/models/user.rb

```diff
@@ -64,6 +64,12 @@ def accessible_grids
   end
 
   ##
+  # @param [Grid] grid
+  def has_access?(grid)
+    self.accessible_grids.include?(grid)
```

Comment:
Nice benchmark :1st_place_medal: 

Surprising that `user.grids.find(...)` is even slower than `user.grids.include?(...)`, it would be interesting to see what queries Mongoid does under the hood. OFC the `user.grids.find(...)` will also return the Grid object...